tests/ViewTest.php

```diff
@@ -117,4 +120,27 @@ public function testENoNeedToEscape() {
 		$result = $this->view->e('script');
 		$this->assertEquals('script', $result);
 	}
+
+    public function testNormalizePath(): void
+    {
+        $viewMock = new class extends View {
+            public static function normalizePath(string $path, string $separator = DIRECTORY_SEPARATOR): string
+            {
+                return parent::normalizePath($path, $separator);
+            }
+        };
+
+        self::assertSame(
```

Comment:
Not a huge deal but the rest of the file uses `$this->` instead of `self::` One thing I've learned that is really important is to keep the codebase as consistent as possible so it doesn't look like "Oh that's definitely a spot fadrian06 coded in". When I code in this framework, even though it drives me nuts, I keep all the class vars lowercase. I would much rather do `$Router = Flight::router();` but in order to keep it consistent, I do `$router = Flight::router();` simply because it's keeping it consistent.

Reply:
oh sorry, I hadn't noticed